test/src/test/testtool/TransactionHandling/scenario02.properties

```diff
@@ -17,6 +17,6 @@ ignoreContentBetweenKeys.interruptedException.key2=</message>
 replaceRegularExpressionKeys.msgWas.key1=<message>null</message>
 replaceRegularExpressionKeys.msgWas.key2=<message>REPLACED</message>
 
-#TOMCATBTM/TOMCAT6
+#TOMCAT6
```

Comment:
Still `TOMCAT6`? Shall we update this comment to `TOMCAT9` for now? 😉 

(And soon perhaps 10 or 11)

Reply:
A version number does not make sense in this property file. Removing it.